prompt2model/model_trainer/base.py

```diff
@@ -25,6 +25,28 @@ def __init__(self, pretrained_model_name: str):
         )
         self.wandb = None
 
+    # we can also have another method called register_trainer
+    # or something like this
```

Comment:
Feel like this is unnecessary unless you have a particular reason for it.

```suggestion
```

Reply:
Will remove this, all of those comments were some of my personal issues / todos but will follow the standards